src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,49 @@
 import React from 'react';
+import { Post, User } from '../../types/types';
+import { CommentList } from '../CommentList';
+import usersFromServer from '../../api/users';
+import commentsFromServer from '../../api/comments';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+function findUser(arrayUsers: User[], postId: number) {
+  const user: User | undefined = arrayUsers.find((
```

Comment:
Parameter name `arrayUsers` is not clear, change it to `usersArray` or simply `users` for better readability.